core/frontend/src/Viewport.ts

```diff
@@ -2236,13 +2245,16 @@ export abstract class Viewport implements IDisposable {
     }
 
     if (overridesNeeded) {
+      this._frameStatsCollector.beginTime("featureSymbologyTime");

```

Comment:
Not useful - just assigns to a variable.

Reply:
Removed.